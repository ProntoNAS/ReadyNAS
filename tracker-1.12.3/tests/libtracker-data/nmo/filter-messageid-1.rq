SELECT ?messageId
WHERE { ?x nmo:messageId ?messageId }
ORDER BY ?messageId
