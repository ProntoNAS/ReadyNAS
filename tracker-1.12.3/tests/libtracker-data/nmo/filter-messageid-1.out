"stringly data for nmo:messageId"
